[2025-06-01T16:52:56.200+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_bash_operator.runme_2 manual__2025-06-01T16:52:45+00:00 [queued]>
[2025-06-01T16:52:56.208+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_bash_operator.runme_2 manual__2025-06-01T16:52:45+00:00 [queued]>
[2025-06-01T16:52:56.208+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2025-06-01T16:52:56.235+0000] {taskinstance.py:1382} INFO - Executing <Task(BashOperator): runme_2> on 2025-06-01 16:52:45+00:00
[2025-06-01T16:52:56.246+0000] {standard_task_runner.py:57} INFO - Started process 129 to run task
[2025-06-01T16:52:56.251+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'runme_2', 'manual__2025-06-01T16:52:45+00:00', '--job-id', '15', '--raw', '--subdir', '/home/***/.local/lib/python3.8/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpqlf9bbfi']
[2025-06-01T16:52:56.255+0000] {standard_task_runner.py:85} INFO - Job 15: Subtask runme_2
[2025-06-01T16:52:56.328+0000] {task_command.py:416} INFO - Running <TaskInstance: example_bash_operator.runme_2 manual__2025-06-01T16:52:45+00:00 [running]> on host 5ae1f21badcd
[2025-06-01T16:52:56.440+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='runme_2' AIRFLOW_CTX_EXECUTION_DATE='2025-06-01T16:52:45+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-01T16:52:45+00:00'
[2025-06-01T16:52:56.448+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-06-01T16:52:56.450+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'echo "example_bash_operator__runme_2__20250601" && sleep 1']
[2025-06-01T16:52:56.474+0000] {subprocess.py:86} INFO - Output:
[2025-06-01T16:52:56.476+0000] {subprocess.py:93} INFO - example_bash_operator__runme_2__20250601
[2025-06-01T16:52:57.477+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-06-01T16:52:57.499+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, execution_date=20250601T165245, start_date=20250601T165256, end_date=20250601T165257
[2025-06-01T16:52:57.554+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-06-01T16:52:57.578+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
